js/js.translator/testData/reservedWords/cases/SHOULD_BE_ESCAPED.txt

```diff
@@ -8,6 +8,7 @@ false
 for
 if
 in
+interface
```

Comment:
This file is autogenerated. As for the reason of the change, 'interface' is
a reserved word in ES6, and now it's also a keyword in Kotlin (previously
it wasn't).

On Mon, Apr 27, 2015 at 12:32 PM, Andrey Breslav <notifications@github.com>
wrote:

> In js/js.translator/testData/reservedWords/cases/SHOULD_BE_ESCAPED.txt
> <https://github.com/JetBrains/kotlin/pull/652#discussion_r29135991>:
>
> > @@ -8,6 +8,7 @@ false
> >  for
> >  if
> >  in
> > +interface
>
> Why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JetBrains/kotlin/pull/652/files#r29135991>.
>